### DISCUSSION TITLE
First release?

### DISCUSSION THREAD

**Discussion by 00lex** [Motion-Project/motionplus#6] [General]
Hi im accidentally found your new project. I was a big fan of motion but switched to pir sensors and own scripts because its hard outside with wind, spiders etc. to find one config for all. then I try a tone of other cameras and have no luck. 
now I see this and im very interested because of opencv. 

its too early but do you have any plan when you release first alpha/beta? 
ah! and will there be a armhf version of motion plus?

thank you

**Comment by Mr-Dave** [comment]
First, thanks for using the Discussions rather than Issues.

Next, it will be some time before this is released as a version.  It compiles right now and I am running it on my system but there are definitely many issues that need to be resolved before it is release ready.  

For the OpenCV, I've not had the best experience yet.  Motionplus does execute the OpenCV process for cascade testing and it seems to work but getting a good cascade model for it has been a challenge.  For the spider and wind issues I added the standard deviation and threshold ratio methods but have not had an opportunity to refine them and add them as part of the default primary detection method.  

With respect to arm builds, I will say that I am moving away from that processor.  If someone else wants to run MotionPlus  on such a under powered platform, that is up to them.  Motionplus will be intended to run on more robust systems that can handle such things as OpenCV processing.

**Comment by 00lex** [comment]
it is sad to read how you feel about arm. the original motion (behind motioneye) runs very well on raspberrys. 
and with every rpi generation better and better. 
I thought the arm platform gets more wind with the apple m1. and I even hope because of that more happens with arm. 

is that your firm decision to distance yourself from arm? 
for me its like in 2000 when I bann CDs or 2013 when I bann mechanical drives from my life. 2021 is the year to bann x86 :) 

i'm waiting hopefully for a release of motion+. in the meantime, exciting new hardware from the raspberry foundation will surely appear :)
